packages/@uppy/transloadit/src/index.ts

```diff
@@ -251,9 +275,9 @@ export default class Transloadit<
 
   client: Client<M, B>
 
-  activeAssemblies: Record<string, Assembly>
+  assembly: Assembly
```

Comment:
Do we want to keep this public?

Reply:
I need it in tests. And since we no longer save the assembly response in state but inside `Assembly`, I think it's fine to allow people to access it.

Reply:
Maybe let's mark it as `readonly`, so it's clear it's an internal field?

Reply:
If I do, then I get an error everywhere where we actually set `assembly`.

Reply:
Too bad TS doesn't support this use case, it'd be useful to have an `internal` visibility